tests/main/snap-mgmt/task.yaml

```
@@ -74,6 +74,9 @@ prepare: |
         test "$(find /etc/systemd/system -name 'snap.*.slice' | wc -l)" -gt 0
     fi
 
+remove: |
```

this should be restore, no ?
```suggestion
restore: |
```

```
Format checks failed for task ./tests/main/snap-mgmt/task.yaml
 - key 'remove' is not among the supported keys: ['summary', 'details', 'backends', 'systems', 'manual', 'priority', 'warn-timeout', 'kill-timeout', 'environment', 'prepare', 'restore', 'debug', 'execute']
```